cfy_manager/components/restservice/restservice.py

```diff
@@ -319,6 +323,21 @@ def _generate_admin_password_if_empty(self):
             config[MANAGER][SECURITY][ADMIN_PASSWORD] = \
                 self._generate_password()
 
+    @staticmethod
+    def _is_in_cluster_mode():
+        return (len(config[SERVICES_TO_INSTALL]) == 1 and
+                config[SERVICES_TO_INSTALL][0] == MANAGER_SERVICE)
```

Comment:
this can be written as just `config[SERVICES_TO_INSTALL] == [MANAGER_SERVICE]`

Reply:
Right